manageiq-operator/pkg/helpers/miq-components/httpd.go

```diff
@@ -142,14 +142,12 @@ func HttpdAuthConfigMap(cr *miqv1alpha1.ManageIQ, scheme *runtime.Scheme) (*core
 	return configMap, f
 }
 
-func PrivilegedHttpd(authType string) (bool, error) {
+func PrivilegedHttpd(authType string) bool {
 	switch authType {
 	case "internal", "openid-connect":
-		return false, nil
+		return false
 	case "external", "active-directory", "saml":
-		return true, nil
-	default:
-		return false, fmt.Errorf("unknown authentication type %s", authType)
+		return true
```

Comment:
I think you still need a default case, otherwise the compiler can't know what the return type is if you don't have one of these Strings, since it only knows "string-in-bool-out" (or is there some default return value in Go?)

Reply:
Done.